download/index.markdown

```diff
@@ -169,6 +169,7 @@ the filename need to be duplicated to `/etc/xbps.d/`.
   * https://mirrors.dotsrc.org/voidlinux/ (EU: Denmark)
   * http://www.gtlib.gatech.edu/pub/VoidLinux/ (USA: Atlanta)
   * https://void.webconverger.org/ (APAN: Singapore)
+  * https://youngjin.io/voidlinux (ASIA: South Korea)
```

Comment:
region should be APAN

Reply:
I had to google it so I was not sure. I will change that on the docs also.